packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
+### Application Feature object
+
+Defines the application feature.
+
+#### Application Feature example
+```json
+{
+    "name": "USER_PROVISIONING",
+    "status": "ENABLED",
+    "description": "User provisioning settings from Okta to a downstream application",
+    "capabilities": {
+        "create": {
+            "lifecycleCreate": {
+                "status": "DISABLED"
+            }
+        },
+        "update": {
+            "profile": {
+                "status": "DISABLED"
+            },
+            "lifecycleDeactivate": {
+                "status": "DISABLED"
+            },
+            "password": {
+                "status": "DISABLED",
+                "seed": "RANDOM",
+                "change": "KEEP_EXISTING"
+            }
+        }
+    },
+    "_links": {
+        "self": {
+            "href": "http://rain.okta1.com:1802/api/v1/apps/0oa1h43s0sZyK6Ft40g4/features/USER_PROVISIONING",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+#### Application Feature properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | Default |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- |
+| capabilities            | capabilites     | [Capabilities Object](#capabilties-object)                                                                  | FALSE    | FALSE   | TRUE    |           |
+| description            | description      | String  | FALSE    | FALSE   | TRUE    |           |
+| _links           | discoverable resources related to the app user     | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)     | TRUE    | FALSE   | TRUE    |           |
+| name              | identifiying name     | `USER_PROVISIONING`  | FALSE    | FALSE  | TRUE    |           |       |      |
+| status            | status    | `ENABLED`, `DISABLED`    | FALSE    | FALSE   | TRUE    |      `DISABLED`  |
+
+##### Capabilties object
+
+The Capabilities object determines the capabilities of an app feature.
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| create            | determines whether the Okta creates or links a user in the application when assigning the app to a user in Okta | [Create Object](#create-object)  | TRUE    | FALSE   | FALSE    |
```

Comment:
> Determines whether Okta creates a new user or links to an existing user in the application when assigning the app to a user in Okta
(I don't know if that's correct, but the current description doesn't make sense).